inc/item_remotemanagement.class.php

```diff
@@ -0,0 +1,370 @@
+      }
+      $canedit = $item->canEdit($ID);
+
+      if ($canedit
+          && !(!empty($withtemplate) && ($withtemplate == 2))) {
+         echo "<div class='center firstbloc'>".
+               "<a class='vsubmit' href='".self::getFormURL()."?itemtype=$itemtype&items_id=$ID&amp;withtemplate=".
+                  $withtemplate."'>";
+         echo __('Add a remote management');
+         echo "</a></div>\n";
+      }
+
+      echo "<div class='center'>";
+      $iterator = self::getFromItem($item);
+
+      $rand = mt_rand();
+      if ($canedit && count($iterator)) {
+         Html::openMassiveActionsForm('mass'.__CLASS__.$rand);
+         $massiveactionparams
+            = ['num_displayed'
+                        => min($_SESSION['glpilist_limit'], count($iterator)),
+                     'container'
+                        => 'mass'.__CLASS__.$rand];
+         Html::showMassiveActions($massiveactionparams);
+      }
+
+      echo "<table class='tab_cadre_fixehov'>";
+      $colspan = 9;
+      echo "<tr class='noHover'><th colspan='$colspan'>".self::getTypeName(count($iterator)).
+            "</th></tr>";
+
+      if (count($iterator)) {
+         $header = '<tr>';
+         $header .= "<th width='10'>".Html::getCheckAllAsCheckbox('mass'.__CLASS__.$rand);
+         $header .= "</th>";
+         $header .= "<th>".__('Remote ID')."</th>";
+         $header .= "<th>".__('Type')."</th>";
+         $header .= "<th>".__('Automatic inventory')."</th>";
+         $header .= "</tr>";
+         echo $header;
+
+         Session::initNavigateListItems(
+            __CLASS__,
+            //TRANS : %1$s is the itemtype name,
+            //        %2$s is the name of the item (used for headings of a list)
+            sprintf(
+               __('%1$s = %2$s'),
+               $item::getTypeName(1),
+               $item->getName()
+            )
+         );
+
+         $mgmt = new self();
+         while ($data = $iterator->next()) {
+            $mgmt->getFromResultSet($data);
+            echo "<tr class='tab_bg_2'>";
+
+            echo "<td width='10'>";
+            Html::showMassiveActionCheckBox(__CLASS__, $data["id"]);
+            echo "</td>";
+            echo "<td>".$mgmt->getRemoteLink()."</td>";
+            echo "<td>".$mgmt->fields['type']."</td>";
+            echo "<td>".Dropdown::getYesNo($data['is_dynamic'])."</td>";
+            echo "</tr>";
+            Session::addToNavigateListItems(__CLASS__, $data['id']);
+         }
+         echo $header;
+      } else {
+         echo "<tr class='tab_bg_2'><th colspan='$colspan'>".__('No item found')."</th></tr>";
+      }
+
+      echo "</table>";
+      echo "</div>";
+   }
+
+
+   /**
+    * Get remote management system link
+    *
+    * @return string
+    */
+   public function getRemoteLink(): string {
+      $link= '<a href="%s" target="_blank">%s</a>';
+      $id = $this->fields['remoteid'];
+      $href = null;
+      switch ($this->fields['type']) {
```

Comment:
Is there a case missing for `LiteManager` ?

Reply:
Nope, I did not find anything